docs/hazmat/primitives/padding.rst

```diff
@@ -54,6 +54,47 @@ multiple of the block size.
             provider.
 
 
+.. class:: ANSIX923(block_size)
+
+    ANSI X.923 padding works by appending ``N-1`` bytes with the value of ``0``
```

Comment:
I can put the link to wikipedia:
https://en.wikipedia.org/wiki/Padding_%28cryptography%29#ANSI_X.923